### ISSUE TITLE
Docker 4 Windows UI should be screen reader accessible

### ISSUE THREAD

**Issue by StefanScherer** [docker/for-win#2084]
From Docker Community Slack team:

Hi guys! a friend of mine who happens to be a blind Microsoft MVP. Yep no mistake there! He is blind and he is having trouble with the desktop experience. He have issues accessing the settings tabs cause the "labels" of those settings are not available to screen readers such as NVDA (https://www.nvaccess.org). He knows there is a slide bar he can adjust but no clue if he is changing CPU or memory preferences. Do you have a team for accesibility? if you want to get in touch with him this is his twitter handler: @kastwey Thanks!!! (edited)

### Expected behavior

UI texts and elements should be readable with screen readers like NVDA

### Actual behavior

They aren't.

### Steps to reproduce the behavior

  1. Install free version of NVDA
  2. Try out to read UI elements

**Comment by rn** [comment]
Thanks for reporting. I don't know how complex it is to fix this but I've created an internal issue so someone can take a look

**Comment by StefanScherer** [comment]
I tried NVDA with Docker for Windows 18.05.0-ce-win66 and the Speech viewer. So there are already a lot of texts accessible. 

<img width="1086" alt="Settings dialog when Windows containers are active" src="https://user-images.githubusercontent.com/207759/40891590-a70784e8-6788-11e8-98f6-4eb2557353c7.png">

After switching to Linux containers the Settings have more options.
And here I understand the problem. The sliders don't show the name of the options, but only "slider xx" and a number.

<img width="1102" alt="Advanced Settings when Linux containers are active, here are unnamed sliders" src="https://user-images.githubusercontent.com/207759/40891633-900da6cc-6789-11e8-8a4a-060c69585766.png">

**Comment by cmendible** [comment]
Thanks for looking into this!

**Comment by cmendible** [comment]
Still open, please /remove-lifecycle stale

**Comment by mat007** [comment]
A fix should has been included in v18.06.1-ce-win74-edge and should reach the next stable release.

**Comment by cmendible** [comment]
Hey @kastwey can you check the edge version?

**Comment by StefanScherer** [comment]
I have tried Docker Community Edition 20.0.0-beta1-win75 Channel edge.

<img width="1440" alt="docker for windows advanced settings, slider from 0 to 100" src="https://user-images.githubusercontent.com/207759/47279400-1ad75180-d5d1-11e8-833d-d554abdbb918.png">

On the left you can see the NVDA speech viewer output.
What I can read here is still suboptimal for a blind person.
All the sliders go from 0 to 100 (percent), but not from the minimum to maximum of the specific setting.

My machine has 4 CPUs, but the slider has values 0, 33, 66 and 100. It still seems confusing to me.
The slider should have values 1, 2, 3 and 4.

Same for memory, slider should have the same value as shown in the text above: 1024 ... 5888

**Comment by LoreenRH** [comment]
Hi @StefanScherer,

That seems to be NVDA that normalizes whatever it gets from the WPF sliders to percentage.

See https://github.com/nvaccess/nvda/issues/1535

**Comment by StefanScherer** [comment]
Thanks @LoreenRH for the link. Then everything looks fine 👍

**Comment by LoreenRH** [comment]
I'll go ahead and close this ✨ thanks for the feedback.
